packages/utils/src/anr.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * A node.js watchdog timer
+ * @param pollInterval The interval that we expect to get polled at
+ * @param anrThreshold The threshold for when we consider ANR
+ * @param callback The callback to call for ANR
+ * @returns
+ */
+export function watchdogTimer(pollInterval: number, anrThreshold: number, callback: () => void): () => void {
```

Comment:
Can we leave this in the node package for now until another package needs access to it?